pkgs/poetry/poetry-in-venv.nix

```diff
@@ -1,23 +1,32 @@
 { pkgs, python, pypkgs, version, url, sha256 }:
-pkgs.stdenv.mkDerivation {
-  name = "poetry-in-venv";
-  inherit version;
+let
+  poetry = pkgs.stdenv.mkDerivation {
+    name = "poetry-in-venv";
+    inherit version;
 
-  src = builtins.fetchTarball {
-    inherit url sha256;
-  };
-
-  buildInputs = [ pypkgs.pip ];
+    src = builtins.fetchTarball {
+      inherit url sha256;
+    };
```

Comment:
Removed pip as a buildInput here. Didn't need it anyway and it was conflicting with something.

Reply:
oooooh nice i think this unblocks the python modules being moved to nixpkgs-unstable